.ado/templates/e2e-test-job.yml

```diff
@@ -20,27 +20,6 @@ jobs:
         clean: true
         fetchDepth: 1 # the depth of commits to ask Git to fetch
         submodules: false
-
-      - task: PowerShell@2
-        displayName: List disksize after checkout
```

Comment:
Could we leave this top task in case we start running out of memory again?

Reply:
Gah.. AutoMerge strikes again

Reply:
The prepare env task already lists the disk space